### DISCUSSION TITLE
4.29 社区周会

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#508] [General]
晚8
录像在 https://www.bilibili.com/video/BV1wY4y1b7a5?spm_id_from=333.999.0.0

议题：
### name 和type解耦 @zhenjunMa
https://github.com/mosn/layotto/issues/513

待认领

### 跨云部署、在线实验室 需求评审 @seeflood
交互设计 v0.2 见：
https://github.com/mosn/layotto/issues/471

action: 先按 v0.2 实现，后续再考虑流量治理的事情 @seeflood 

### 跨云部署 -- File API 改造 @wenxuwan 
#510

本次会议action: 
- 提个 PR，支持 s3 API @wenxuwan 

### transaction mesh @stulzq 先看下
https://developer.aliyun.com/article/786297

### CI 改造、容器镜像改造 @Xunzhuo
#483 
- 可以给大家演示下新 CI
- demo 测试会不会和 文档测试有冲突；考虑做可移植性测试

本次会议action:  
- pr review、合并 @seeflood 
- 裁剪分享视频，上传 b站、贴到官方文档里 @zhenjunMa 
- 合并后重构文档测试、添加可移植性测试 @seeflood 

### wasm 进展同步（拿 wasm 写组件）
上次action: 关于拿 wasm 写组件，我们需要个 POC   ,即使POC 结论是“做不了”也算有进展，可以找Wasmer社区提需求。  @rayowang 跟进。

本次会议action: WASI 没提供封装好的socket api，如果我们要做的话，需要把 https://github.com/WebAssembly/WASI/pull/312  这个 pr 里的代码粘贴过来，风险是遇到问题的话很难查。 @rayowang  和 @zhenjunMa  讨论下

### secret API 功能完善
- 可能涉及不兼容变更，需要讨论决策
#472 

## 2.新需求评审
https://github.com/mosn/layotto/issues/500

本次会议action:  
- 给出一个设计方案 @seeflood 

## 3. v0.4 发版事项review
https://github.com/mosn/layotto/milestone/2

本次会议action:  
- @rayowang 和 @akkw 认领修复bug的issue
- 修复完成后 release v0.4
## 4. 自由讨论
某司落地实践
![image](https://user-images.githubusercontent.com/26001097/166085677-29f91f10-b68b-4e49-b359-84958e14aab3.png)

讨论了集成 jaeger 和prometheus 遇到的问题，等 CI 重构合并后，再补充 prometheus 的quickstart; @LXPWing 
其他任何想聊的，比如开发遇到的问题

**Comment by khotyn** [comment]
Great!

**Comment by seeflood** [comment]
社区周会欢迎所有人, 要参会的朋友可以来钉钉群哈
大家平时打字沟通效率不高、互相回复也不及时，有任何问题都可以来会议聊，高效沟通~
![image](https://user-images.githubusercontent.com/26001097/165898482-a4a672d3-d1b5-46c9-b458-2cb0d8649886.png)
